src/CMakeLists.txt

```diff
@@ -132,6 +132,8 @@ set(DBB-BOOTLOADER-SOURCES ${DBB-BOOTLOADER-SOURCES} PARENT_SCOPE)
 
 set(DRIVER-SOURCES
   ${CMAKE_SOURCE_DIR}/src/peripherals_init.c
+  ${CMAKE_SOURCE_DIR}/src/platform/driver_init.c
+  ${CMAKE_SOURCE_DIR}/src/qtouch/qtouch.c
```

Comment:
hmmm shouldn't it go to external/? the file does no seem to be written by us:

> Copyright (c) 2017 Microchip. All rights reserved.

Reply:
about half-half I would say. Since we have written code and put in "technically" external files. Both are anyway generated code which is meant to be changed. So it is a tough case. Since they depend heavily on the hardware platform they have to be where we can control the macros.

Reply:
Makes sense :+1: